spec/models/local_transaction_spec.rb

```diff
@@ -0,0 +1,105 @@
+RSpec.describe LocalTransaction do
+  let(:content_store_response) do
+    GovukSchemas::Example.find("local_transaction", example_name: "local_transaction_with_devolved_administration_availability")
+  end
+
+  it "gets the introduction" do
```

Comment:
Could we do this and the following two methods like this: 

```
describe "#introduction" do
  it "returns the data from the content item" do (etc)
```

Reply:
Sure. Will update and fix up.

Reply:
I've gone with

```ruby
describe "#introduction" do
    it "gets the introduction from the content store response" do
```
As `content_item` is being used in the tests to refer to the model.